apps/nova/src/components/TutorProfile/TutorLoadingError.tsx

```diff
@@ -0,0 +1,24 @@
+import ExclaimationMarkCircle from "@litespace/assets/ExclaimationMarkCircle";
+import { Button, ButtonVariant } from "@litespace/luna/Button";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { Typography } from "@litespace/luna/Typography";
+import { Void } from "@litespace/types";
+
+export const TutorLoadingError: React.FC<{ refetchRatings: Void }> = ({
+  refetchRatings,
+}) => {
+  const intl = useFormatMessage();
+  return (
+    <div className="flex flex-col mt-20 gap-10 items-center justify-center">
+      <div className="p-[6px] bg-destructive-200 rounded-full">
+        <ExclaimationMarkCircle />
+      </div>
```

Comment:
The width and hight of this element doesn't match the design. Please don't let this details miss you. 


You can control the width and the height of the svg by updating it [here](https://github.com/litespace-org/litespace/blob/master/packages/assets/assets/exclaimation-mark-circle.svg?short_path=bbf6e89#L1).

Set the width and hight to `100%` 

```html
<svg width="100%" height="100%" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
    ...
</svg>
```

Then in react, you can do this. 

```tsx
<ExclaimationMarkCircle width={64} height={64} />;
```


Note that this will break the other places that are using `ExclaimationMarkCircle`. Make sure to update these places as well.